composer.json

```diff
@@ -20,7 +20,7 @@
         "psr/container": "^1.0",
         "symfony/http-foundation": "^3.1 || ^4.0",
         "symfony/http-kernel": "^2.7 || ^3.0 || ^4.0",
-        "symfony/property-access": "^2.7 || ^3.0 || ^4.0",
+        "symfony/property-access": "^2.7 || ^3.0 <3.4.19 || ^4.0 <4.1.8",
```

Comment:
as there has been a fix, shouldn't these be declared as conflict ?

Reply:
I'll just remove this a 4.2 is here anyway

Reply:
Yeah but these 2 versions still conflicts somehow. Or you can categorize them as "yup doesn't work we know" ?